EveripediaNetwork/ep-ui
# Rankpage Search Functionality

# Rankpage Search Functionality

This PR:
- adds search functionality to all the tabs on the rankapage which allow users to easily search for tokens, stablecoins, nfts and other rank items.
- improves the rank pagination UX by hiding the prev/next buttons on the first/last page respectively in order to avoid showing negative table entries on few occasions

## How should this be tested?

1. Go to the rank page
2. Search for any item within the table across all 5 tabs
3. Use the pagination buttons to ensure negative table entries are not shown as well.


## Linked issues

closes https://github.com/EveripediaNetwork/issues/issues/3039
